### ISSUE TITLE
Ability to subscribe to just the first post of an announcement

### ISSUE THREAD

**Issue by sharplet** [thoughtbot/constable#807]
I'm a big fan of the feature in [Discourse](https://www.discourse.org) that for a given category, you can choose to just watch for new posts, but not replies. This gives me an opportunity to see what's being discussed and decide what I'd like to participate in before I subscribe to an entire thread.

**Comment by tysongach** [comment]
This sounds somewhat ideal for our `hiring` Interest - I tend to unsubscribe from those announcements because they get tons of rapid comments. That said, those announcements also have the `everyone` Interest on them, so I’m not sure it would make any difference.

This also reminded me of https://github.com/thoughtbot/constable/issues/430

**Comment by sharplet** [comment]
Yeah if we tied this to interests, we'd need to figure out which notification level would be adopted based on the combination of the notification levels for each interest. Having said that, I'd probably subscribe to `everyone` at first-post level too.

**Comment by tysongach** [comment]
This feels slightly better than #430 because I think you’d just need to configure it once, whereas a “temporary unsubsribe” requires user action each time.
